crates/swc/tests/tsc-references/inlineJsxAndJsxFragPragma.1.normal.js

```diff
@@ -29,12 +29,15 @@ export { };
 /*#__PURE__*/ jsx(React.Fragment, null);
 //// [preacty-no-fragment.tsx]
 /**
+ * @jsx h
+ * @jsxFrag Fragment
+ */ /**
```

Comment:
A duplicate comment generation issue has been identified, and I'm plan fixes this in next PRs.

Reply:
Are you fine using `dev` branch while this issue is fixed?

Reply:
Fixed